conf/test_rnaspades.config

```
@@ -26,6 +26,7 @@ params {
 
     skip_eukulele = true
     skip_eggnog   = true
+    skip_eukulele = true
```

```suggestion
    skip_kofamscan = true
```
Probably also fix spaces...